src/app/plot.py

```diff
@@ -0,0 +1,221 @@
+    world["name"] = world["name"].apply(lambda x:str.lower(" ".join(x.split(" ")[0:2])))
+    
+    world = world.loc[world["name"] != "antarctica"]
+    source = data[["Country", "Salary_USD"]].groupby("Country").median().reset_index()
+    source["Country"] = source["Country"].apply(lambda x:str.lower(x))
+    source.rename({"Country":"name"}, axis=1, inplace=True)
+
+    datamap = pd.merge(world, source, how='left')
+    datamap['Salary_USD'] = datamap['Salary_USD'].fillna(0)
+    
+    chart = alt.Chart(datamap).mark_geoshape().encode( 
+        color=alt.Color(field = "Salary_USD",type = "quantitative",
+                        scale=alt.Scale(type = "sqrt"),
+                        legend=alt.Legend(title="Salary in USD",labelFontSize = 10,symbolSize = 10,titleFontSize=10)),
+        tooltip=['name:N', 'Salary_USD:Q', "alpha:Q"]
+    )
+    
+    if xmax is not None:
+        datamap["alpha"] = 1
+        datamap.loc[datamap["name"] == xmax.lower(), "alpha"] = 100
+        chart = chart.encode(
+            opacity=alt.Opacity(field = "alpha",type = "quantitative", scale=alt.Scale(type = "sqrt")),
+        )
+        
+    chart = chart.properties(
+        title='Median Salary of The World',
+        projection={"type":'mercator'},
+        width=600,
+        height=525,
+    ).configure_axis(
+        labelFontSize=10
+    )    
+    
+    return chart.to_html()
+
+def plot_13(DS_identity=['Yes', 'No', 'Sort of (Explain more)'], df=data.copy()):
+    # Clean data
+    df = df.dropna()
+    df = df.query("Salary_USD < 400_000")
+    df = df[df["Tenure"] != "I don't write code to analyze data"]
+    
+    # Filter data
+    if DS_identity == None:
+        DS_identity = ['Yes', 'No', 'Sort of (Explain more)']
+    if not isinstance(DS_identity, list):
+        DS_identity = list(DS_identity)
+    df = df[df['DataScienceIdentitySelect'].isin(DS_identity)]
+
+    # alt.themes.enable("dark")
+
+    # Create Plot
+    brush = alt.selection_interval()
+    click = alt.selection_multi(fields=["Tenure"])
+
+    points = (
+        alt.Chart(df, title="Select a window for interactive coding experience count")
+        .mark_circle()
+        .encode(
+            y=alt.Y("Country", title=None),
+            x=alt.X("Salary_USD", title="Salary in USD"),
+            color=alt.condition(
+                brush,
+                alt.Color("Tenure:N", legend=None),
+                alt.value("lightgray"),
+            ),
+            opacity=alt.condition(click, alt.value(1.0), alt.value(0.1)),
+            tooltip="EmployerIndustry",
+        )
+        .add_selection(brush)
+    ).properties(
+        width=300,
+        height=680
+    )
+
+    bars = (
+        alt.Chart(df, title="Click to filter the above plot!")
+        .mark_bar()
+        .encode(
+            x="count()",
+            y=alt.Y("Tenure", title="Coding Experience", sort=['More than 10 years', '6 to 10 years', '3 to 5 years', '1 to 2 years', 'Less than a year']),
```

Comment:
It seems misleading to rename `Tenure` as `Coding Experience`

Reply:
@artanzand